documentation/src/main/asciidoc/query.asciidoc

```diff
@@ -339,12 +368,12 @@ Query luceneQuery = mythQB
 ----
 ====
 
-The above query returns results containing "storm" and "tree" either in "name", "history" or "description".
+The above query returns results containing `storm` and `tree` either in `name`, `history` or `description`.
 
-The syntax supported by the SimpleQueryParser includes the following operators: boolean (AND, OR, NOT),
+The syntax supported by the `SimpleQueryParser` includes the following operators: boolean (AND `+`, OR `|`, NOT `-`),
```

Comment:
I was just changing it following the above remark of Sanne (and the use of a list to present each option). :)